src/helpers.js

```diff
@@ -80,8 +80,27 @@ var insertTask = function(namespace, input, options) {
   });
 };
 
+var getModelData = function({query, limit, continuation, key, Model}, callback) {
+  return Model.query(query, {
+    limit,
+    continuation,
+  }).then(function(data) {
+    var retval = {};
+    retval[key] = data.entries.map(function(entry) {
+      return entry.json();
+    });
+    if (data.continuation) {
+      retval.continuationToken = data.continuation;
+    }
+    return callback(null, retval);
+  });
+};
+
 // Export insertTask
 exports.insertTask = insertTask;
 
+// Export getModelData
+exports.getModelData = getModelData;
```

Comment:
By "list function, then export statement following it" @jonasfj means that the `export` statement for each function should come directly after the definition of that function.

Reply:
`export` statement is now directly after the definition of `listTableEntries`. 
https://github.com/taskcluster/taskcluster-index/pull/39/files#diff-b38f7254be1a9f817fd2e33c294481a1R114